### PR TITLE
LPS-183217 Add indexes to object relationship many-to-many dynamic tables

### PR DESCRIPTION
**Pull request by mateussgn**
Related issue: https://issues.liferay.com/browse/LPS-183217

### CODE REVIEW

**Comment by mateussgn** [comment]
I'll send the integration test in fallow up PR, it's already in development.

**Comment by mateussgn** [comment]
ci:test:sf

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:test:upgrade

**Review by guilhermedcamacho — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-service/bnd.bnd [hunk 1]
```diff
@@ -2,6 +2,6 @@ Bundle-Name: Liferay Object Service
 Bundle-SymbolicName: com.liferay.object.service
 Bundle-Version: 1.0.151
 Liferay-Client-Extension-Batch: com/liferay/object/internal/batch
-Liferay-Require-SchemaVersion: 5.1.1
+Liferay-Require-SchemaVersion: 5.2.0
```
Comment (mateussgn) [inline modules/apps/object/object-service/bnd.bnd:5]: @guilhermedcamacho, This change should be made?

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:test:object

**Comment by mateussgn** [comment]
ci:test:upgrade

**Comment by mateussgn** [comment]
ci:test:sf

**Comment by guilhermedcamacho** [comment]
ci:stop

**Review by guilhermedcamacho — APPROVED** [review]
LGTM! :tada:

**Comment by guilhermedcamacho** [comment]
ci:test:relevant

**Comment by guilhermedcamacho** [comment]
ci:test:object

**Comment by guilhermedcamacho** [comment]
ci:test:upgrade

**Comment by mateussgn** [comment]
ci:close

**Pull request closed without merging**